### ISSUE TITLE
link TACC account login to Arctos persephone_s

### ISSUE THREAD

**Issue by mvzhuang** [ArctosDB/arctos#9408]
Sorry dusty, one more. Can we link local persephone_s account to TACC account. Same username across both. This should be a easier issue I think.

**Comment by dustymc** [comment]
@mvzhuang  https://arctos.database.museum/AdminUsers.cfm?action=edit&username=persephone_s is a local account, they should be able to delete it from https://arctos.database.museum/myArctos.cfm, then authenticate with TACC to make a new account that can be made into an Operator.

**Comment by mvzhuang** [comment]
oh perfect. ok done thanks!
